.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 __pycache__/
 migrations/
+db_*
```

Comment:
use '.sqlite3'
I dont think it will match on db.sqlite3 because its missing '_'
Also if we had an app with db in it, it could cause problems

Reply:
I decided to remove that part entirely since apparently we have to remove db.sqlite3 from our repo before the gitignore even applies.